drivers/docker/driver.go

```diff
@@ -572,13 +578,6 @@ START:
 	return recoverableErrTimeouts(startErr)
 }
 
-// nextBackoff returns appropriate docker backoff durations after attempted attempts.
-func nextBackoff(attempted int) time.Duration {
```

Comment:
potential lazy route: maybe some of my "is it the same?" questions above could be resolved by keeping this function, but having _it_ call the new helper?